src/scripts/on.ts

```diff
@@ -92,24 +99,28 @@ declare const GARDEN_VERSION: string;
     });
   };
 
-  const components: { id: string; version: string }[] = [];
+  chrome.runtime.onMessage.addListener((request: { latestVersion: string }) => {
+    const components: { id: string; version: string }[] = [];
 
-  inspect(document, components);
+    inspect(document, components, request.latestVersion);
 
-  const iframes = document.getElementsByTagName('iframe');
+    const iframes = document.getElementsByTagName('iframe');
 
-  for (const iframe of iframes) {
-    if (iframe.contentDocument) {
-      inspect(iframe.contentDocument, components);
-    } else {
-      console.log('Garden Inspect is unable to access cross-origin iframe:', iframe);
+    for (const iframe of iframes) {
+      if (iframe.contentDocument) {
+        inspect(iframe.contentDocument, components, request.latestVersion);
+      } else {
+        console.log('Garden Inspect is unable to access cross-origin iframe:', iframe);
+      }
     }
```

Comment:
I think in the future, we can simplify the extension to be only the service worker:

```
chrome.scripting.executeScript(
    {
      target: {tabId: tabId, allFrames: true},
      func: isOn? off : on ,
    }
);
```
and remove the for loop for the accessing the frames.

Reply:
Cool. Let's add this to the backlog @Francois-Esquire

Reply:
Will do!